server/api/lib/tags/housing.js

```diff
@@ -0,0 +1,36 @@
+// Housing tag applies if the change to either housing by area or housing by units meets the minimum threshold
+const { isTagByUsageAddition} = require('../tags/utils');
+
+const isHousing = async (planId, tag) => {  
+	const HOUSING_BY_AREA = 'housingByArea';
+	const HOUSING_BY_UNITS = 'housingByUnits';
+	const planTags = [];
+	const dataRules = [];
+	
+	// check housing by area
+	const housingByArea = await isTagByUsageAddition(planId,tag,HOUSING_BY_AREA); 
+	if (housingByArea) {
+		dataRules.push(
+			housingByArea.created_by_data_rules
+		);
+	}
+	// check housing by units
+	const housingByUnits = await isTagByUsageAddition(planId,tag,HOUSING_BY_UNITS);
+	if (housingByUnits) {
+		dataRules.push(
+			housingByUnits.created_by_data_rules
+		);
+	}
+	if (housingByArea || housingByUnits)
+		planTags.push( {
+			plan_id: planId,
+			tag_id: tag,
+			display_score: 0, /* TODO: Add the correct display score here */
+			created_by_data_rules: `[${dataRules.toString(',')}]`
+		});
+	return planTags;
+}
+
+module.exports = {
```

Comment:
General structure for tags:

We have a folder of tags= housing, employment, commerce, etc. 
Each file (housing.js) should export one function- `doesApply` or `doesTagApply` or any name you find suitable. Then, we have an index.js file with all tags, their label (as done in constants.js, but in our tag folder) and the isApply function.

Reply:
Then, the code that will run the tag-extraction can access all rules using the index.js file, and shouldn't know it explicitly

Reply:
Sounds good. I refactored it based on this feedback in https://github.com/meirim-org/meirim/pull/389/commits/7583ef1a2cfe5292725020c9e3e41870d8323ba8 PLMK if it looks good!